tests/common/mod.rs

```diff
@@ -0,0 +1,188 @@
+use bitcoincore_rpc::{bitcoin::Network, json, RpcApi};
+use bitcoind::{BitcoinD, Conf, ConnectParams};
+use flate2::read::GzDecoder;
+use std::env;
+use std::process::{Child, Command, Stdio};
+use std::thread;
+use tar::Archive;
+use tempfile::{tempdir, Builder, TempDir};
+use tokio::time::Duration;
+
+const LND_VERSION: &str = "0.16.0";
+
+// setup_test_infrastructure spins up all of the infrastructure we need to test LNDK, including a bitcoind node and two
+// LND nodes. LNDK can then use this test environment to run.
+pub async fn setup_test_infrastructure() -> (BitcoinD, LndNode, TempDir) {
+    let (bitcoind, bitcoind_dir) = setup_bitcoind().await;
+    let lnd_exe_dir = download_lnd().await;
+
+    let lnd_node = LndNode::new(bitcoind.params.clone(), lnd_exe_dir);
+    lnd_node.setup_client().await;
+
+    return (bitcoind, lnd_node, bitcoind_dir);
+}
+
+pub async fn setup_bitcoind() -> (BitcoinD, TempDir) {
+    let bitcoind_dir = tempdir().unwrap();
+    let bitcoind_dir_path = bitcoind_dir.path().clone().to_path_buf();
+    let mut conf = Conf::default();
+    conf.tmpdir = Some(bitcoind_dir_path);
+    conf.args = vec![
+        "-regtest",
+        "-fallbackfee=0.0001",
```

Comment:
Oh I only added that because it's one of the default values in bitcoind https://github.com/RCasatta/bitcoind/blob/master/src/lib.rs#LL161C36-L161C47 But seems like for our purposes it's unnecessary. :thinking: